app/views/interview/index.html.erb

```diff
@@ -5,6 +6,28 @@
 		<td>Interviewer Name</td>
 		<td>Datetime</td>
 		<td>Location</td>
+		<td>Cancel</td>
+	</tr>
+	<% @my_interviews.each do |i| %>
+	<tr>
+		<td><%= link_to i.id, interview_path(i) %></td>
+		<td><%= i.applicant_name %></td>
+		<td><%= i.interviewer_name %></td>
+		<td><%= i.time %></td>
+		<td><%= i.location %></td>
+		<td><%= link_to "Cancel", interview_cancel_path(i.id) %></td>
+	</tr>
+	<% end %>
+</table>
+<h2>All Interviews</h2>
```

Comment:
Might be a good place for a partial!

On Wed, Jul 15, 2015 at 2:34 PM, Stephen Ou <notifications@github.com>
wrote:

> In app/views/interview/index.html.erb
> <https://github.com/ASESStanford/MemberApp/pull/6#discussion_r34732418>:
>
> > @@ -5,6 +6,28 @@
> >  		<td>Interviewer Name</td>
> >  		<td>Datetime</td>
> >  		<td>Location</td>
> > +		<td>Cancel</td>
> > +	</tr>
> > +	<% @my_interviews.each do |i| %>
> > +	<tr>
> > +		<td><%= link_to i.id, interview_path(i) %></td>
> > +		<td><%= i.applicant_name %></td>
> > +		<td><%= i.interviewer_name %></td>
> > +		<td><%= i.time %></td>
> > +		<td><%= i.location %></td>
> > +		<td><%= link_to "Cancel", interview_cancel_path(i.id) %></td>
> > +	</tr>
> > +	<% end %>
> > +</table>
> > +<h2>All Interviews</h2>
>
> Yeah that's a good idea. I'll move it to /interview/admin.
>
> I put everything together originally to eliminate duplicate views since
> they all look the same.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ASESStanford/MemberApp/pull/6/files#r34732418>.
>



-- 
Regards,
*David Jiang *
*Class of 2016 *

*(510) 996-2119 | LinkedIn
<http://www.linkedin.com/pub/david-jiang/36/215/a73> *